src/AlternateLife.RageMP.Net/Data/CommandInformation.cs

```diff
@@ -1,20 +0,0 @@
-using AlternateLife.RageMP.Net.Interfaces;
-using AlternateLife.RageMP.Net.Scripting;
-
-namespace AlternateLife.RageMP.Net.Data
-{
-    internal class CommandInformation
```

Comment:
Is this class so different to `Command`? You could still use it but edit properties to fit your needs